Source/Scene/Fog.js

```diff
@@ -131,7 +131,7 @@ define([
 
         // Fade fog in as the camera tilts toward the horizon.
         var positionNormal = Cartesian3.normalize(camera.positionWC, scratchPositionNormal);
-        var dot = CesiumMath.clamp(Cartesian3.dot(camera.directionWC, positionNormal), 0.0, 1.0);
+        var dot = Math.abs(Cartesian3.dot(camera.directionWC, positionNormal));
```

Comment:
I changed this because I noticed that any views that are looking from horizon-view to birds-eye-view would result in a negative dot product. So all views between horizon and birds-eye are considered to be in complete fog, but really it should adjust.